editions/resumebuilder/tiddlers/Résumé Wizard.tid

```diff
@@ -1,7 +1,7 @@
 created: 20150719173441992
-modified: 20150719182326153
+modified: 20150720171426440
 tags: 
-title: Resume Wizard
+title: Résumé Wizard
```

Comment:
One trick that we're missing is to use GitHub commit hooks to check some of this stuff....